src/platform-includes/getting-started-config/apple.mdx

```diff
@@ -25,6 +25,10 @@ func application(_ application: UIApplication,
     [SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
         options.dsn = @"___PUBLIC_DSN___";
         options.debug = YES; // Enabled debug when first installing is always helpful
+
+        // Enable tracing to capture 100% of transactions for performance monitoring.
+        // If you want to choose the sampling rate, you can use 'options.tracesSampleRate'
```

Comment:
In other SDKs we usually mention something like "We recommend adjusting this value in production" for `tracesSampleRate` to help prevent users from running out of quota.

Maybe we could say something like:

```suggestion
        // Enable tracing to capture 100% of transactions for performance monitoring.
        // Use 'options.tracesSampleRate' to set the sampling rate. 
        // We recommend setting a sample rate in production.
```